### PR TITLE
Add Gray Matter theme

### PR DESCRIPTION
**Pull request by philipbelesky**
Hey, you probably don't need another 'minimal' Markdown theme, but I thought I'd put this out there just in case. 

To my mind 'writer' skips a few nice details (like muting syntax) and handles the sidebar strangely (dark blue color + black highlight state). 

FWIW the colours/approach here are from a [slightly-popular](https://github.com/philipbelesky/gray-matter) theme of mine for Sublime/VSCode.

### CODE REVIEW

File: Themes/Gray Matter (Dark).theme.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+    "name": "Writer",
+    "style": "Dark",
+    "author": [
```
Comment (DivineDominion) [inline Themes/Gray Matter (Dark).theme.json]: You should be able to add any field that's not yet taken and provide a link to your VSCode theme, if you want, like `"basedOn" : "https://github.com/philipbelesky/gray-matter"` or something. Either at the project root, or as a note in the author subcategory, or whatever suits you best :)

**Comment by philipbelesky** [comment]
Good point, I'll make a note to get back to this in a week - I should probably take a bit more time with it.

**Comment by philipbelesky** [comment]
Hey just a quick follow up that this should be finalised now and ready to merge. Thanks for the offer to accept it!

**Comment by DivineDominion** [comment]
Thanks a lot! I'll merge in the changes and test drive the theme for a while :)

**Pull request merged**